cpematcher/core.py

```diff
@@ -23,14 +23,6 @@ class CPE:
         "other",
     ]
 
-    @property
-    def matches_all(self):
```

Comment:
not sure we want to break an existing api

Reply:
I don't see the need and no idea what is the use-case for that, as the `.matches()` method is considering this case too, so I'm not seeing where this would be called instead of `.matches()`